src/main/kotlin/lotto/Main.kt

```diff
@@ -0,0 +1,7 @@
+package lotto
+
+import lotto.controller.LottoController
+
+fun main() {
+    LottoController.run()
+}
```

Comment:
Please activate the configuration to prevent EOF alerts on GitHub. 🙂 

Editor -> General -> Ensure every saved file ends with a line break